ONSdigital/rm-collection-exercise-service
# Validation exception changes with tests

# Motivation and Context


# What has changed

Changed the way the CE event validation process handles exceptions and returns data


Made changes to unit tests to accommodate changes

# How to test?
run unit tests
Manual test with invalid information in resops (accompanying PR) and check correct error message is displayed.

ASAIK tried every possible type of invalid information submitted through the form manually, unit tests in ResOps for 'categories' of errors.

https://trello.com/c/tVq25lkO/791-fix-error-messaging-when-validating-the-ce-event-create-update-forms


ResOps PR: https://github.com/ONSdigital/response-operations-ui/pull/301
